recipes/nspr/all/conanfile.py

```diff
@@ -0,0 +1,165 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from contextlib import contextmanager
+import os
+
+
+class NsprConan(ConanFile):
+    name = "nspr"
+    homepage = "https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSPR"
+    description = "Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level and libc-like functions."
+    topics = ("conan", "nspr", "libc")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    license = "MPL-2.0"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_mozilla": [True, False],
+        "win32_target": ["winnt", "win95"]
```

Comment:
I am not sure if it's still useful to provide Windows 95 support, but if it's easy, I am okay with that

Reply:
I've not tested it as I have no appropriate toolchain, but it was just a matter of a configuration argument.
So maybe we make somebody happy with it...